tutorials/users/catalog/pystac-client.ipynb

```diff
@@ -0,0 +1,1929 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "277e13cf-d858-450d-8134-7cbf4b2b0623",
+   "metadata": {},
+   "source": [
+    "# Catalog access via python client"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5fc75ef1-9335-4505-9e5c-ee1d4600b586",
+   "metadata": {},
+   "source": [
+    "The GUI search discussed in the previous section is great for (i) quickly exploring the catalog to see what data is \n",
+    "available on the network, (ii) for examining the properties of various collections, and (iii) for quickly getting access links to \n",
+    "small number of files. If your analysis is limited to using selected data files, then you could hardcode the access links to \n",
+    "those files that you obtained from the STAC Browser.\n",
+    "\n",
+    "However, often you might need several data files from a collection \n",
+    "and manually searching for and copy-pasting the links to those files from the Browser into your notebooks is not feasible, \n",
+    "and a programmatic approach becomes immediately desirable. A more serious limitation of hardcoding access links from the Browser \n",
+    "is that if the links for the data change, then your code will not function unless you manually update the code with the \n",
+    "new links (again obtained from a manual search in the STAC Browser). This can lead to your code becoming less reproducible \n",
+    "and can create problems for you down the line and problems for anyone with whom you've shared the code. For these reasons \n",
+    "we recommend that you limit the GUI interface to the first two aforementioned use cases, and for accessing data links, you \n",
+    "use the programmatic approach described here."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "613014df-a557-41f1-a22c-3159e2bd2979",
+   "metadata": {},
+   "source": [
+    "## pystac-client"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "94cdba30-fa1c-4a6d-9e34-6b296b6d45ad",
+   "metadata": {},
+   "source": [
+    "The library that we will use to programmatically interact and query the Marble STAC catalog is called [`pystac-client`](https://pystac-client.readthedocs.io/en/stable/). This is built on the robust [`pystac`](https://pystac.readthedocs.io/en/stable/) library for creating and reading STAC Catalogs, \n",
+    "and extends that by allowing the ability to work with a STAC API. We highly encourage you to look at the tutorials on the `pystac-client` \n",
```

Comment:
> [`pystac`](https://pystac.readthedocs.io/en/stable/)
Take the backticks out of the link

Reply:
I'm not sure I follow. Could you explain a bit more?

Reply:
I believe @alexandercyu just means that ``[`pystac-client`]`` should be `[pystac-client]`

Reply:
I have been using the convention of code syntax for library names. Unless there is objection to doing that site-wide then I think we can leave this as is.

Reply:
Just a note for the future... 

We should make a style guide for this sort of thing so that we can make sure we're consistent across all tutorials

Reply:
The pystac-client link format has backticks in it and it renders like  the other backtick text.  It's inconsistent formatting and it doesn't immediately show that it's a link

Reply:
To me this paragraph renders like: 

![image](https://github.com/DACCS-Climate/marble-tutorials/assets/4380924/ca560055-44fa-42a1-9682-6fbd0819e859)

So we get the backtick version with the link (blue text) vs. the backtick version without the link pink-ish text.

@alexandercyu are you saying that we should remove the link or the backticks?
And do you mean it is inconsistent with other links or other backticked text?

Reply:
I'm saying just remove the backticks.  It's as you pointed out: ``[`pystac-client`]`` should be `[pystac-client]`

Reply:
> To me this paragraph renders like:
> 
> ![image](https://private-user-images.githubusercontent.com/4380924/300609054-ca560055-44fa-42a1-9682-6fbd0819e859.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY2MzA5NTcsIm5iZiI6MTcwNjYzMDY1NywicGF0aCI6Ii80MzgwOTI0LzMwMDYwOTA1NC1jYTU2MDA1NS00NGZhLTQyYTEtOTY4Mi02ZmJkMDgxOWU4NTkucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDEzMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAxMzBUMTYwNDE3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZmNiMTYzODIxYTQ5YTk4ZTMzZTEwMmFlYjcyMmQ4NWNlYWJmZGE0YTVhNDRjODQ1NWFiODFkYmQxZTQ4MDcwNyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.M5wUfDqzdbDE02rCrscuebtM7n4wgVONpYCxFOlu-GU)
> 
> So we get the backtick version with the link (blue text) vs. the backtick version without the link (pink-ish text).
> 
> @alexandercyu are you saying that we should remove the link or the backticks? And do you mean it is inconsistent with other links or other backticked text?

I don't know what happened the first time I rendered, but it is now rendering for me like you have shown.  I recall seeing the backtick link and the non-backtick link rendering the same.  But now that this is resolved I think it's good to go.

Reply:
Thanks for the clarification @alexandercyu. I was wondering if there was a compilation issue you were facing...